src/Lykke.MatchingEngine.Connector/Services/TcpOrderSocketService.cs

```diff
@@ -37,22 +37,31 @@ public void HandleDataFromSocket(object data)
             try
             {
                 switch (data)
-                {
-                    case TheResponseModel theResponse:
-                        _logger?.Add($"Response ProcessId: {theResponse.ProcessId}");
-                        _tasksManager.SetResult(theResponse.ProcessId, theResponse);
-                        break;
-                    case TheNewResponseModel theNewResponse:
-                        _logger?.Add($"Response Id: {theNewResponse.Id}");
-                        _newTasksManager.SetResult(theNewResponse.Id, theNewResponse);
-                        break;
-                    case MarketOrderResponseModel theMarketOrderResponse:
-                        _logger?.Add($"Response Id: {theMarketOrderResponse.Id}");
-                        _marketOrdersTasksManager.SetResult(theMarketOrderResponse.Id, theMarketOrderResponse);
-                        break;
-                    case MeMultiLimitOrderResponseModel multiLimitOrderResponse:
-                        _logger?.Add($"Response Id: {multiLimitOrderResponse.Id}");
-                        _multiOrdersTasksManager.SetResult(multiLimitOrderResponse.Id, multiLimitOrderResponse);
+                {
+                    case TheResponseModel theResponse:
+                        _logger?.Add($"Response ProcessId: {theResponse.ProcessId}");
+                        _tasksManager.SetResult(theResponse.ProcessId, theResponse);
+                        break;
+                    case TheNewResponseModel theNewResponse:
+                        _logger?.Add($"Response Id: {theNewResponse.Id}");
+                        _newTasksManager.SetResult(theNewResponse.Id, theNewResponse);
+                        break;
+                    case MarketOrderResponseModel theMarketOrderResponse:
+                        _logger?.Add($"Response Id: {theMarketOrderResponse.Id}");
+                        _marketOrdersTasksManager.SetResult(theMarketOrderResponse.Id, theMarketOrderResponse);
+                        break;
+                    case MeMultiLimitOrderResponseModel multiLimitOrderResponse:
+                        _logger?.Add($"Response Id: {multiLimitOrderResponse.Id}");
+                        _multiOrdersTasksManager.SetResult(multiLimitOrderResponse.Id, multiLimitOrderResponse);
+                        break;
+                    // No handlers for the following messages
+                    case MePingModel m0: 
+                    case MeCashInOutModel m1:
+                    case MeLimitOrderModel m2:
+                    case MeLimitOrderCancelModel m3:
+                    case MeUpdateBalanceModel m4:
+                    case MeUpdateWalletCredsModel m5:
+                    case MeNewMarketOrderModel m6:
                         break;

```

Comment:
Let's add here:
_logger?.Add($"{data.ToJson()}");

Reply:
These messages are expected but not have handlers. For unexpected messages exists the default section. Logging can affect performance (we can spam out azure) without obvious advantages